acceptance/tests/facts/partitions.rb

```diff
@@ -13,7 +13,7 @@
       ['uuid', /^[-a-zA-Z0-9]+$/],
       ['partuuid', /^[-a-f0-9]+$/],
       ['mount', /^\/.*/],
-      ['label', /\w+/],
```

Comment:
This will match anything now ("zero or more word characters anywhere in the string"). Do we want to instead just validate that the fact exists, and not try to enforce any format on it (since volume labels are freeform anyway)